### PR TITLE
framework/iotbus : make wrappers for each component and clean the source codes

### PR DESCRIPTION
**Pull request by jeongarmy**
- make wrappers for each component, i2c, gpio, pwm, spi, and uart
- apply coding rule and clean the source codes

### CODE REVIEW

**Comment by seinfra** [comment]
Code Rule Check Result:
framework/src/iotbus/iotbus_gpio.c:91: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_gpio.c:96: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_i2c.c:64: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_i2c.c:69: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_pwm.c:82: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_pwm.c:87: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_spi.c:92: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_uart.c:86: ERROR: [SPC_M_KWD] space required before the open parenthesis '('
framework/src/iotbus/iotbus_uart.c:91: ERROR: [SPC_M_KWD] space required before the open parenthesis '('

**Comment by seinfra** [comment]
Code Rule Check OK.

File: framework/src/iotbus/iotbus_uart.c [hunk 1]
```diff
@@ -67,31 +71,44 @@ int _iotbus_valid_baudrate(unsigned int rate)
...
 
 int iotbus_uart_stop(iotbus_uart_context_h hnd)
 {
-	if (!hnd)
+	if (!hnd || !hnd->handle) {
 		return IOTBUS_ERROR_INVALID_PARAMETER;
+	}
 
-	close(hnd->fd);
+	close(hnd->handle->fd);
```
Comment (sangwon03) [inline framework/src/iotbus/iotbus_uart.c]: please use local handler for code readability

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request merged**